smoketest/regions/dev/hosts.yaml

```diff
@@ -0,0 +1,15 @@
+---
+aro_clusters:
+  hosts:
+    aro-adenton:
```

Comment:
nothing against you, :) but can this be made to be more generic?

Reply:
Oops yes, I did that but missed one place :D